core/dbt/events/eventmgr.py

```diff
@@ -214,6 +214,14 @@ def add_logger(self, config: LoggerConfig):
         logger.event_manager = self
         self.loggers.append(logger)
 
+    def add_callbacks(self, callbacks: List[Callable[[EventMsg], None]]):
+        """Helper for bulk adding a sequence of Callbacks to the EventManager"""
+        self.callbacks.extend(callbacks)
+
+    def add_callback(self, callback: Callable[[EventMsg], None]):
```

Comment:
Do we need this function?

Reply:
We do not need it, no. I included it as syntactic sugar for anyone who has a use case wherein they know they are only adding one callback or have to add callbacks one at a time due to having to track some extra context whenever adding a callback. Essentially when adding only one callback
```
EVENT_MANAGER.add_callback(singular_callback)
```
is more idiomatic than having people do
```
EVENT_MANAGER.add_callbacks([singular_callback])
```
Functionally though there is no difference :shrug:

Reply:
Dropped in https://github.com/dbt-labs/dbt-core/pull/7214/commits/9bceac887084a248e3f5d52612aa928af9e00569